templates/docker-compose.template

```diff
@@ -81,17 +76,6 @@ services:
       io.balena.features.procfs: 1
       io.balena.features.dbus: 1
 
-  dbus-session:
-    image: balenablocks/dbus:rpi-0.0.2
-    restart: always
-    volumes:
-      - dbus:/session/dbus
-    environment:
-      - DBUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
-      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
-      - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
-
 volumes:
   miner-storage:
```

Comment:
Do we still need miner-storage volume?

Reply:
No, I had removed it forgot to stage the change.

Reply:
Actually, we probably need to leave this in for cases where people are using a data-only hotspot and want the file based private key to be persistent

Reply:
checked all container again, hm-diag uses it for permanent storage.

Reply:
Yes and hm-gatewayrs should also use it for persistent key storage for file based key

Ref https://github.com/NebraLtd/hm-gatewayrs/issues/28